packages/api-client/src/api/getCart/index.ts

```diff
@@ -29,7 +30,7 @@ const emptyCart: Cart = {
 
 export default async function getCart({ client, config }: ApiContext): Promise<Cart> {
   try {
-    const token = await getCurrentBearerOrCartToken({ client, config });
+    const token = await getCurrentBearerOrCartToken({ client, config }) as RequiredAnyToken;
```

Comment:
The `getCurrentBearerOrCartToken` function returns an `OptionalAnyToken ` type which technically might be an empty object that `RequiredAnyToken` does not accept. We might need a better way of handling that.